tests/IntegrationTests/IntegrationTestCase.php

```diff
@@ -0,0 +1,32 @@
+<?php declare(strict_types=1);
+
+namespace Lmc\Matej\IntegrationTests;
+
+use Lmc\Matej;
+
+class IntegrationTestCase extends Matej\TestCase
+{
+    protected function setUp(): void
+    {
+        parent::setUp();
```

Comment:
Maybe use eg. `init()` method with `/** @before */` annotation, to avoid calling the parent::setUp?